File: tests/fixtures/sample.java
import java.util.List;
import java.util.ArrayList;

public class Animal {
    private String name;

    public Animal(String name) {
        this.name = name;
    }

    public String speak() {
        return "";
    }

    public String getName() {
        return name;
    }
}

class Dog extends Animal {
    public Dog(String name) {
        super(name);
    }

    public String speak() {
        return "Woof";
    }
}

File: tests/fixtures/sample.py


class Animal:
    def __init__(self, name: str):
        self.name = name

    def speak(self) -> str:
        return ""


class Dog(Animal):
    def speak(self) -> str:
        return "Woof"

    def fetch(self, item: str) -> bool:
        return True


def greet(name: str) -> str:
    return f"Hello, {name}"


def add(a: int, b: int) -> int:
    return a + b
